### PR TITLE
Issue #14436: Enforced naming convention for  `NestedForDepthCheckTest`

### PR DESCRIPTION
**Pull request by abhishekmaity**
Part of #14436

### CODE REVIEW

**Comment by abhishekmaity** [comment]
@romani @nrmancuso PR ready for submission

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/nestedfordepth/InputNestedForDepthLevelFour.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
  * @author Alexander Jesse
  * @see com.puppycrawl.tools.checkstyle.checks.coding.NestedForDepthCheck
  */
-public class InputNestedForDepth1 {
+public class InputNestedForDepthLevelFour {
```
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/nestedfordepth/InputNestedForDepthLevelFour.java:16]: InputNestedForDepthLevelFour --> `InputNestedForDepthCheckCustomMaxLevelFour `

The word "Check" needs to be added by convention for all input files of src/test. Please read Issue description at #14436

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/nestedfordepth/InputNestedForDepthLevelTwo.java [hunk 2]
```diff
@@ -13,7 +13,7 @@
  * @author Alexander Jesse
  * @see com.puppycrawl.tools.checkstyle.checks.coding.NestedForDepthCheck
  */
-public class InputNestedForDepth {
+public class InputNestedForDepthLevelTwo {
```
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/nestedfordepth/InputNestedForDepthLevelTwo.java:16]: InputNestedForDepthLevelFour --> `InputNestedForDepthCheckCustomMaxLevelTwo `

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/NestedForDepthCheckTest.java [hunk 3]
```diff
@@ -68,17 +68,17 @@ public void testNestedTooDeep() throws Exception {
...
 
         verifyWithInlineConfigParser(
-                getPath("InputNestedForDepth1.java"),
+                getPath("InputNestedForDepthLevelFour.java"),
                expected);
     }
 
     @Test
-    public void testTokensNotNull() {
+    public void testNestedForDepthTokensNotNull() {
```
Comment (MANISH-K-07) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/NestedForDepthCheckTest.java:81]: Please revert this for now. We are only changing test methods that extract java input files in this issue

**Review by MANISH-K-07 — COMMENTED** [review]
@abhishekmaity ,
Please do me a favor....
[InputNestedForDepthDefault.java](https://github.com/checkstyle/checkstyle/blob/master/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/nestedfordepth/InputNestedForDepthDefault.java) --> `InputNestedForDepthCheckDefaultMaxLevel`

Also update test method names according to reviewed suggestions of filenames

**Comment by abhishekmaity** [comment]
@MANISH-K-07 suggestion implemented

**Comment by abhishekmaity** [comment]
@romani @nrmancuso PR ready for submission after revision

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/NestedForDepthCheckTest.java [hunk 4]
```diff
@@ -44,7 +44,7 @@ protected String getPackageLocation() {
      *     interface-requirements for test-methods.
      */
     @Test
-    public void testNestedTooDeep() throws Exception {
+    public void testNestedForDepthLevelTwo() throws Exception {
```
Comment (MANISH-K-07) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/NestedForDepthCheckTest.java:47]: Please do rename the test method name according to new inputs file name
testNestedForDepthLevelTwo --> `testNestedForDepthCheckCustomMaxLevelTwo`

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/NestedForDepthCheckTest.java [hunk 5]
```diff
@@ -68,12 +68,12 @@ public void testNestedTooDeep() throws Exception {
      *     interface-requirements for test-methods.
      */
     @Test
-    public void testNestedOk() throws Exception {
+    public void testNestedForDepthLevelFour() throws Exception {
```
Comment (MANISH-K-07) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/NestedForDepthCheckTest.java:71]: Same here

File: src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/NestedForDepthCheckTest.java [hunk 6]
```diff
@@ -92,13 +92,13 @@ public void testTokensNotNull() {
     }
 
     @Test
-    public void testNestedDefault() throws Exception {
+    public void testNestedForDepthDefault() throws Exception {
```
Comment (MANISH-K-07) [inline src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/NestedForDepthCheckTest.java:95]: Same here

**Review by MANISH-K-07 — COMMENTED** [review]
Last Items :

**Review by MANISH-K-07 — COMMENTED** [review]
Great.... Everything lgtm now

**Review by nrmancuso — APPROVED** [review]
@abhishekmaity thanks for improving this test suite!

**Pull request merged**